packages/devtools_app/lib/src/inspector/flutter/inspector_screen.dart

```diff
@@ -13,6 +13,7 @@ import '../../flutter/screen.dart';
 import '../../flutter/split.dart';
 import '../../globals.dart';
 import '../../service_extensions.dart' as extensions;
+import '../../ui/fake_flutter/_real_flutter.dart';
```

Comment:
nit: remove this? I think you should be able to remove this

Reply:
`import 'package:flutter/foundation.dart';` would be the correct import

Reply:
good catch! replaced with the flutter/foundation import